Lib/tracemalloc.py

```diff
@@ -280,7 +291,7 @@ def size(self):
 
     @property
     def traceback(self):
-        return Traceback(self._trace[2])
+        return Traceback(self._trace[2], self._trace[3])
```

Comment:
Since total_nframe can be None, maybe now it would make more sense to handle Python 3.8 compatibility on demand in this property:

```
# Call Traceback(frames, total_nframe).
# For a Python 3.8 trace, call Traceback(frames).
return Traceback(*self._trace[2:])
```

(and revert change in `_Traces.__init__`)

Well, you wrote something like that in a previous PR :-)

Reply:
Good idea!